README.md

```diff
@@ -27,4 +27,9 @@ To start the server run `lapis server` and visit http://localhost:8080 to see th
 
 The code cache is currently turned off, so refreshing the page will show any changes to the code immediately. 
 
-Submit changes via PR, and happy hacking!
\ No newline at end of file
+Submit changes via PR, and happy hacking!
+
+# Translation
+We are using the i18n module by kikito for internationalization. https://github.com/kikito/i18n.lua
+
+In the static/i18n folder in the project there are several .lua files for i18n translations. If you look in en.lua all of the string keys which are in use through the site are listed, and adding the corresponding key to the de.lua file for instance will add the German translation. Then when the locale is loaded it will load the entries from the language which matches the locale set by the user. If the key isn't found in that locale (en-US for instance) then it will move to the parent locale (en in the example case). If it still doesn't find an entry for that key, it will look in the default (en in our case) and load that. TODO: Figure out a better process around i18n translations. 
```

Comment:
Mention that we'd like to eventually use Crowdin for it like we do right now

Reply:
adjusted the todo to reflect that